client/.eslintrc.js

```diff
@@ -145,6 +145,7 @@ module.exports = {
     'no-underscore-dangle': 'off',
     'object-curly-newline': ['error', { consistent: true }],
     'prefer-destructuring': 'off',
+    'no-restricted-exports': 'off',
```

Comment:
Without this, ESLint will always warn if we do exports like in the re-exporting `index.ts` files, i.e., `export { default } from './DayCalendar';`.

I think this is a rule from Airbnb's, and the point of this rule is to prevent modules name-exporting reserved keywords, but I do not see a particular use case in our code base. Also, `default` is not necessarily considered a named export, and this was discussed [here, too](https://github.com/airbnb/javascript/issues/2500).